paparazzi/paparazzi-gradle-plugin/src/test/projects/test.settings.gradle

```diff
@@ -4,4 +4,13 @@ dependencyResolutionManagement {
       from(files("../../../../../gradle/libs.versions.toml"))
     }
   }
+
+  repositories {
```

Comment:
What do you think about adding "fail on project repos" or "prefer ..." to ensure that there's no mixing between settings.gradle repos and build.gradle repos, it's either-or.

Reply:
default is [PREFER_PROJECT](https://github.com/gradle/gradle/blob/d62830774ae867342c1b2b0e73f7575df953aa26/subprojects/dependency-management/src/main/java/org/gradle/internal/management/DefaultDependencyResolutionManagement.java#L85), which seems fine to me, as some test projects may want custom libraries to assert expected behaviors.